### PR TITLE
[MLDM-140] return fileset when inspectSnapshot; fix bazel 

### PR DESCRIPTION
**Pull request by Zhang-Muyang**

### CODE REVIEW

**Review by brendoncarroll — APPROVED** [review]
LGTM

**Review by jrockway — APPROVED** [review]
Looks good.

File: src/internal/snapshot/pretty/pretty.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ func PrintDetailedSnapshotInfo(info *snapshot.SnapshotInfo) error {
 	t, err := template.New("SnapshotInfo").Funcs(funcMap).Parse(
 		`ID: {{.Id}}
 Chunkset: {{.ChunksetId}}
-Created: {{prettyAgo .Created}}{{if .PachydermVersion}}
+Created: {{prettyAgo .CreatedAt}}{{if .PachydermVersion}}
```
Comment (jrockway) [inline src/internal/snapshot/pretty/pretty.go:25]: Actually, it would be good to show the SQL Dump Fileset here.

Comment (Zhang-Muyang) [reply src/internal/snapshot/pretty/pretty.go:25]: oh makes sense. I'll fix this.

**Pull request merged**